tests/ert/unit_tests/config/observations_generator.py

```diff
@@ -183,6 +183,9 @@ def summary_observations(
         "error_mode": draw(st.sampled_from(ErrorMode)),
         "value": draw(positive_floats),
     }
+
+    assume(kws["error_mode"] == ErrorMode.ABS or kws["error"] < 2)
```

Comment:
This is in a separate commit, but think it has effect on logic from the first commit? If so they should be squashed so the tests pass on all commits.